tests/phpunit/class-test-core-sitemaps.php

```diff
@@ -80,13 +80,18 @@ public function test_core_sitemaps_get_max_urls() {
 		// Apply a filter to test filterable values.
 		add_filter( 'core_sitemaps_max_urls', array( $this, 'filter_max_url_value' ), 10, 2 );
 
-		$this->assertEquals( core_sitemaps_get_max_urls(), CORE_SITEMAPS_MAX_URLS, 'Can not confirm max URL number.' );
-		$this->assertEquals( core_sitemaps_get_max_urls( 'posts' ), 300, 'Can not confirm max URL number for posts.' );
-		$this->assertEquals( core_sitemaps_get_max_urls( 'taxonomies' ), 50, 'Can not confirm max URL number for taxonomies.' );
-		$this->assertEquals( core_sitemaps_get_max_urls( 'users' ), 1, 'Can not confirm max URL number for users.' );
+		$expected_null = core_sitemaps_get_max_urls();
+		$expected_posts = core_sitemaps_get_max_urls( 'posts' );
+		$expected_taxonomies = core_sitemaps_get_max_urls( 'taxonomies' );
+		$expected_users = core_sitemaps_get_max_urls( 'users' );
 
 		// Clean up.
 		remove_filter( 'core_sitemaps_max_urls', array( $this, 'filter_max_url_value' ) );
+
+		$this->assertEquals( $expected_null, CORE_SITEMAPS_MAX_URLS, 'Can not confirm max URL number.' );
```

Comment:
I could use a data provider here if you think it would be cleaner rather than including four insertions. However, since I'm really testing the filter and not the individual values, I thought leaving them all together were more visible.

Reply:
SGTM 👍